lib/mpl_toolkits/axisartist/axes_rgb.py

```diff
@@ -1,8 +1,6 @@
-from mpl_toolkits.axes_grid1.axes_rgb import (
-    make_rgb_axes, imshow_rgb, RGBAxesBase)
-
+from mpl_toolkits.axes_grid1.axes_rgb import make_rgb_axes, imshow_rgb, RGBAxes
 from .axislines import Axes
 
 
-class RGBAxes(RGBAxesBase):
+class RGBAxes(RGBAxes):
```

Comment:
??? auto-relplaced?

Reply:
it's inheriting from the RGBAxes class imported from the other (base) module, which happens *just* above.